netdata.spec.in

```diff
@@ -515,6 +515,9 @@ rm -rf "${RPM_BUILD_ROOT}"
 # freeipmi files
 %attr(4750,root,netdata) %{_libexecdir}/%{name}/plugins.d/freeipmi.plugin
 
+# go.d.plugin (the capability required for wireguard module)
+%caps(cap_net_admin=epi) %{_libexecdir}/%{name}/plugins.d/go.d.plugin
```

Comment:
@Ferroin Do we need to check here if the file exists before doing `caps()` (go.d is optional I guess)? If yes - I don't know how to do it in rpm spec file.

Reply:
No, in the spec files we’re fine to do it unconditionally.

The only way we end up with it being optional would be having a separate package for it (which I think we _should_ do, but that’s orthogonal to this discussion), but in that case that separate package would just handle the capabilities.